app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -100,7 +100,7 @@ class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::To
           :data => {'toggle'        => 'modal',
                     'target'        => '#modal_d_div',
                     'function'      => 'miqCallAngular',
-                    'function-data' => '{"name": "showListener", "args": []}'})
+                    'function-data' => '{\"name\": \"showListener\", \"args\": []}'})
```

Comment:
Where? I mean, I'd understand if the whole thing was a string but here.. The value should not be interpolated anywhere..